src/components/PDFThumbnail/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import {Document, pdfjs, Thumbnail} from 'react-pdf';
+import FullScreenLoadingIndicator from '@components/FullscreenLoadingIndicator';
+import useThemeStyles from '@hooks/useThemeStyles';
+import addEncryptedAuthTokenToURL from '@libs/addEncryptedAuthTokenToURL';
+import type PDFThumbnailProps from './types';
+
+pdfjs.GlobalWorkerOptions.workerSrc = new URL(
+    'pdfjs-dist/legacy/build/pdf.worker.min.js',
+    // @ts-expect-error - It is a recommended step for import worker - https://github.com/wojtekmaj/react-pdf/blob/main/packages/react-pdf/README.md#import-worker-recommended
+    import.meta.url,
+).toString();
```

Comment:
GlobalWorkerOptions should only be set once. We are setting this on PDFView

Reply:
Hmm, good point. What about changing to this
```js
if (!pdfjs.GlobalWorkerOptions.workerSrc) {
    pdfjs.GlobalWorkerOptions.workerSrc = new URL(
        'pdfjs-dist/legacy/build/pdf.worker.min.js',
        // @ts-expect-error - It is a recommended step for import worker - https://github.com/wojtekmaj/react-pdf/blob/main/packages/react-pdf/README.md#import-worker-recommended
        import.meta.url,
    ).toString();
}
```

Reply:
Do you think we can move this to platformSetup if it's not going to slow down the load time

Reply:
That js file is around 1MB so I think it's better to load it when needed.
<img width="359" alt="image" src="https://github.com/Expensify/App/assets/117511920/6f507539-cb24-4863-8cde-bc16bc54ed4f">

Reply:
Then maybe we should load it within a useEffect as defining it outside seems to execute even if we are not loading any pdf

Reply:
Hmm, I think it should be fine to have this without useEffect as we only load it once globally.

Reply:
@s77rt I just find that this method doesn't work when offline. Previous I tested offline case using the offline feature provided by App and today I disabled network by `Console` => `Network` => `Offline`. Do you have the same issue on your client? 

Using the method of PDFView works both online and offline.

Reply:
Yes same issue. Let's use the current method of PDFView. I have reported this bug here https://github.com/Expensify/App/pull/33434#discussion_r1506963546